plugins/nanospell/plugin.js

```diff
@@ -919,17 +911,6 @@
 				this.wrapWithTypoSpan(editor, currRange);
 			}
 		},
-		markAllTypos: function (editor) {
-			var range = editor.createRange(),
-				block;
-
-			range.selectNodeContents(editor.editable());
-
-			var iterator = range.createIterator();
-			while (( block = iterator.getNextParagraph() )) {
-				this.markTypos(editor, block);
-			}
-		},
```

Comment:
Ah yes.  I noticed this function was dead code as well, after the previous thing.